iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractOverloadManagementSystemTest.java

```diff
@@ -70,13 +70,18 @@ public void baseTests() {
         assertEquals(2, substation.getOverloadManagementSystemCount());
 
         Iterable<OverloadManagementSystem> iterable = substation.getOverloadManagementSystems();
-        List<OverloadManagementSystem> retreived = StreamSupport.stream(iterable.spliterator(), false).toList();
-        assertEquals(2, retreived.size());
-        assertTrue(retreived.containsAll(ImmutableSet.of(oms1, oms2)));
-
-        retreived = substation.getOverloadManagementSystemStream().toList();
-        assertEquals(2, retreived.size());
-        assertTrue(retreived.containsAll(ImmutableSet.of(oms1, oms2)));
+        List<OverloadManagementSystem> retrieved = StreamSupport.stream(iterable.spliterator(), false).toList();
+        assertEquals(2, retrieved.size());
+        assertTrue(retrieved.containsAll(Set.of(oms1, oms2)));
+
+        retrieved = substation.getOverloadManagementSystemStream().toList();
+        assertEquals(2, retrieved.size());
+        assertTrue(retrieved.containsAll(Set.of(oms1, oms2)));
+
+        retrieved = network.getIdentifiableStream(IdentifiableType.OVERLOAD_MANAGEMENT_SYSTEM)
+            .map(OverloadManagementSystem.class::cast).toList();
+        assertEquals(2, retrieved.size());
+        assertTrue(retrieved.containsAll(Set.of(oms2, oms1)));
```

Comment:
there was a typo `retreived` -> `retrieved`. also cleaned to use basic Java Set.of instead of Google ImmutableSet.of